tests/core/mempool/test_mempool_manager.py

```diff
@@ -534,3 +535,89 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [sb1]
+
+
+def always(mm: MempoolManager, mi: MempoolItem) -> bool:
+    return True
+
+
+def never(mm: MempoolManager, mi: MempoolItem) -> bool:
+    return False
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("test_for", ["max_cost", "max_fee", "normal_path"])
+@pytest.mark.parametrize("inclusion_filter", [always, never])
+async def test_process_mempool_items(
+    test_for: str, inclusion_filter: Callable[[MempoolManager, MempoolItem], bool]
+) -> None:
+    coin1_amount = uint64(10000000)
```

Comment:
Since you are testing multiple cases in this test, it would be a little easier to read if you defined the expected outputs up here. eg.

```
coin1_child = Coin(coin1.name(), IDENTITY_PUZZLE_HASH, coin1_amount - 42)
```